src/uwtools/tests/test_schemas.py

```diff
@@ -362,6 +367,94 @@ def test_schema_jedi_run_dir(jedi_prop):
     assert "88 is not of type 'string'" in errors(88)
 
 
+# make_hgrid
+
+
+def test_schema_make_hgrid():
+    config = {
+        "execution": {"executable": "make_hgrid"},
+        "run_dir": "/tmp",
+    }
+    errors = schema_validator("make-hgrid", "properties", "make_hgrid")
+    # Basic correctness:
+    assert not errors(config)
+    # All top-level keys are required:
+    for key in ("execution", "run_dir"):
+        assert f"'{key}' is a required property" in errors(with_del(config, key))
+    # Additional top-level keys are not allowed:
+    assert "Additional properties are not allowed" in errors({**config, "foo": "bar"})
+    # config needs at least the grid_type key:
+    assert "'grid_type' is a required property" in errors({"config": {}})
+
+
+def test_schema_make_hgrid_grid_type():
+    # Get errors function from schema_validator
```

Comment:
```suggestion
    # Get errors function from schema_validator.
```
I'm going to nitpick-suggest some punctuation on the comments here...